src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -211,6 +212,14 @@ namespace winrt::TerminalApp::implementation
                 e.Handled(true);
             }
         }
+        if (_currentMode == CommandPaletteMode::ActionMode && key == VirtualKey::Tab)
+        {
+            auto const state = CoreWindow::GetForCurrentThread().GetKeyState(winrt::Windows::System::VirtualKey::Control);
+            if (WI_IsFlagSet(state, CoreVirtualKeyStates::Down))
+            {
+                _keyDownHandler(sender, e);
+            }
+        }
```

Comment:
I think we should probably remove this special case - I think there's a bunch of other similar keystrokes that won't work _exactly_ right, but at least this makes intuitive sense when it doesn't work. 

Sorry if I was confusing before!